### PR TITLE
Testcase to verify noobaa status output

### PR DESCRIPTION
**Pull request by jilju**
OCS-2084 - Verify noobaa status

Verifies:
bug https://bugzilla.redhat.com/show_bug.cgi?id=1764195
bug https://bugzilla.redhat.com/show_bug.cgi?id=1764195

Signed-off-by: Jilju Joy <jijoy@redhat.com>

### CODE REVIEW

**Comment by jilju** [comment]
> PR validation on existing cluster [Job PASSED](https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-trigger-test-pr/1056//testReport).

AWS-IPI

> PR validation on existing cluster [Job PASSED](https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-trigger-test-pr/1056//testReport).

AWS-IPI-3AZ-RHCOS-3M-3W

**Comment by jilju** [comment]
> PR validation on existing cluster [Job PASSED](https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-trigger-test-pr/1056//testReport).
AWS-IPI-3AZ-RHCOS-3M-3W

**Comment by jilju** [comment]
> PR validation on existing cluster [Job PASSED](https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-trigger-test-pr/1058//testReport).

OCS4.2 VMware

**Comment by jilju** [comment]
> PR validation on existing cluster [Job PASSED](https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-trigger-test-pr/1059//testReport).

Skipped because of bugzilla marker

File: tests/manage/mcg/test_verify_noobaa_status.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+@tier1
+@noobaa_cli_required
+@polarion_id('OCS-2084')
+@bugzilla('1799077')
+def test_verify_noobaa_status():
+    """
+    Verify noobaa status output is clean without any errors
+    """
+    contents_count = [
+        'CustomResourceDefinition-5', 'Namespace-1', 'ServiceAccount-1',
```
Comment (vasukulkarni) [inline tests/manage/mcg/test_verify_noobaa_status.py]: I hope the numbers dont change across newer builds.

Comment (ebenahar) [reply tests/manage/mcg/test_verify_noobaa_status.py]: @jilju could you please save these in defaults.py?

Comment (jilju) [reply tests/manage/mcg/test_verify_noobaa_status.py]: Done

File: tests/manage/mcg/test_verify_noobaa_status.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+from ocs_ci.utility.utils import run_async
+from ocs_ci.framework.pytest_customization.marks import (
+    tier1, noobaa_cli_required
+)
+from ocs_ci.framework.testlib import polarion_id, bugzilla
+
+log = logging.getLogger(__name__)
+
+
+@tier1
```
Comment (ebenahar) [inline tests/manage/mcg/test_verify_noobaa_status.py]: any reason to have this is tier1? I think it should be part of tier2

Comment (jilju) [reply tests/manage/mcg/test_verify_noobaa_status.py]: Categorized as tier1 because this is doing basic status check.
Should I change it to tier2?

Comment (ebenahar) [reply tests/manage/mcg/test_verify_noobaa_status.py]: yes please

Comment (jilju) [reply tests/manage/mcg/test_verify_noobaa_status.py]: Done

File: ocs_ci/ocs/defaults.py [hunk 3]
```diff
@@ -35,3 +35,11 @@
 OCS_OPERATOR_NAME = "ocs-operator"
 LOCAL_STORAGE_OPERATOR_NAME = "local-storage-operator"
 LIVE_CONTENT_SOURCE = "redhat-operators"
+
+# Noobaa status output
+NOOBAA_STATUS_CONTENT_COUNT = [
```
Comment (RazTamir) [inline ocs_ci/ocs/defaults.py]: A more pythonic way would be to use a list of dictionaries.
```
NOOBAA_STATUS_CONTENT_COUNT = {
'CustomResourceDefinition': 5, 
... 
}
```

Comment (jilju) [reply ocs_ci/ocs/defaults.py]: Changed to dict.

**Comment by RazTamir** [comment]
Hi @jilju ,
Don't forget to remove the [WIP]

**Pull request merged**